app/models/ticket.rb

```diff
@@ -5,11 +5,16 @@ class Ticket < ApplicationRecord
   validates :entered_gate_id, presence: true
   validate :check_distance
 
+  def used?
+    exited_gate_id.present?
```

Comment:
細かいけど、idが存在するか？よりも降りた改札が存在するか？の方がオブジェクト指向らしくていいかなーと思いました。
（好みのレベル）

```suggestion
    exited_gate.present?
```

Reply:
ご指摘の通り`exited_gate.present?`の方がパッとみた時に何やってるか分かりやすいので変更します！